packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -213,13 +213,15 @@ export function EntitiesSavedStatesExtensible( {
 					{ __( 'Are you ready to save?' ) }
 				</strong>
 				{ additionalPrompt }
-				{ isDirty && (
-					<p>
-						{ __(
-							'The following changes have been made to your site, templates, and content.'
-						) }
-					</p>
-				) }
+				<p>
+					{ isDirty
+						? __(
+								'The following changes have been made to your site, templates, and content.'
+						  )
+						: __(
+								'No changes will be made to your site, templates, and content. Select the items you want to save.'
```

Comment:
I think this is a good change in general. 

Would `"Select the items you wish to save."` be enough or do we need a qualifying statement?

Reply:
+1, really good idea having some descriptive text, but I like the idea of shortening it a little if we can 👍

Reply:
> Would `"Select the items you wish to save."` be enough or do we need a qualifying statement?

Certainly, I think this is sufficient. Updated by [4b1036b](https://github.com/WordPress/gutenberg/pull/57716/commits/4b1036bab2acf90da34b73ee5fc4f2fad0f2d1ee)👍